presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/TestParquetDecimalScaling.java

```diff
@@ -127,7 +267,13 @@ protected void dropTable(String tableName)
 
     protected void assertValues(String tableName, int scale, List<String> expected)
     {
-        String formattedValues = Joiner.on(", ").join(expected.stream().map(s -> format("'%s'", s)).collect(toImmutableList()));
+        String formattedValues = Joiner.on(", ").join(expected.stream().map(s -> format("'%s'", new BigDecimal(s).setScale(scale).toString())).collect(toImmutableList()));
+        assertQuery(format("select format('%%.%df', value) FROM %s", scale, tableName), "VALUES " + formattedValues);
```

Comment:
belongs to prev commit; however, i suggested to change thus to `computeActual` anyway